docs/en/ingest-management/beats-agent-comparison.asciidoc

```diff
@@ -67,7 +67,7 @@ To find out if an integration is GA, see the
 The following table shows the outputs supported by the {agent} in {version}:
 
 
-NOTE: {elastic-defend} has a different output matrix.
+NOTE: {elastic-defend} and APM server have a different output matrix.
```

Comment:
```suggestion
NOTE: {elastic-defend} and APM Server have a different output matrix.
```

Reply:
They capitalize server. Not sure why we don't have a shared attribute for this, but it looks like they hard code it in the APM docs, so let's go with that.